src/data/support-card-desc.csv

```diff
@@ -0,0 +1,41 @@
+"text", "translation"
+"スキルPtボーナスと初期賢さアップ","Skill Pt bonus and initial wisdom increase"
+"やる気効果アップと初期絆ゲージアップ","Motivation bonus and initial bond gauge increase"
+"友情ボーナスとレースボーナス","Friendship and race bonus"
+"やる気効果アップとレースボーナス","Motivation and race bonus"
+"友情ボーナスと初期パワーアップ","Friendship bonus and initial power increase"
+"トレーニング効果アップとヒント発生率アップ","Training bonus and hint appearance rate up"
+"友情ボーナスと得意率アップ","Friendship bonus and speciality rate up"
+"トレーニング効果アップと得意率アップ","Training bonus and speciality rate up"
+"友情ボーナスとヒント発生率アップ","Friendship bonus and hint appearance rate up"
+"根性ボーナスと初期根性アップ","Guts bonus and initial guts increase"
+"スキルPtボーナスとヒント発生率アップ","Skill pt bonus and hint appearance rate up"
+"やる気効果アップと得意率アップ","Motivation bonus and speciality rate up""
```

Comment:
`speciality` typo, multiple other places as well

Reply:
Apparently it's a valid word! But for consistency, could you change these to `specialty`? 👀